Contribute/markdown-reference.md

```diff
@@ -308,6 +308,50 @@ Here are requirements and considerations for include files:
 - As with regular articles, don't share media between include files. Use a separate file with a unique name for each include and article. Store the media file in the media folder that's associated with the include.
 - Don't use an include as the only content of an article.  Includes are meant to be supplemental to the content in the rest of the article.
 
+## Indentation
+
+In Markdown, spaces before the first character of a line determine the line's alignment relative to the preceding lines. Indentation especially influences numbered and bulleted lists to render multiple levels of nesting in a hierarchical or outline format.
+
+To indent text to align with a preceding paragraph or an item in a numbered or bulleted list, use spaces.
+
+The following two examples show how indented paragraphs render based on their relationship to other paragraphs.
+
+```
+1. This is a numbered list example (one space after the period before the letter T).
+   This sentence is indented three spaces.
+   This code block is indented three spaces.
+   
+- This is a bulleted list example (one space after the bullet before the letter T).
+  This sentence is indented two spaces.
+  > [!TIP]
+  > This tip is indented two spaces.
+  - This is a second-level bullet (indented two spaces, with one space after the bullet before the letter T).
+    This sentence is indented four spaces.
+    > This quote block is indented four spaces.
+```
```

Comment:
The example does not match the rendering you are trying to demonstrate. You can use the alternative tilde markdown syntax so that you can embed backticks in the code example. Note - this suggestion does not render properly in GitHub but it will work when merged.
```suggestion
~~~markdown
1. This is a numbered list example (one space after the period before the letter T).
   This sentence is indented three spaces.

   ```
   This code block is indented three spaces.
   ```

- This is a bulleted list example (one space after the bullet before the letter T).
  This sentence is indented two spaces.
  > [!TIP]
  > This tip is indented two spaces.
  - This is a second-level bullet (indented two spaces, with one space after the bullet before the letter T).
    This sentence is indented four spaces.
    > This quote block is indented four spaces.
~~~
```